README.md

```diff
@@ -1,8 +1,8 @@
 [![clang format](https://github.com/webmachinelearning/webnn-native/actions/workflows/clang_format_check.yml/badge.svg)](https://github.com/webmachinelearning/webnn-native/actions/workflows/clang_format_check.yml)
-[![Memory leak check - DirectML backend (Windows)](https://github.com/webmachinelearning/webnn-native/actions/workflows/memory_leak_check_dml.yml/badge.svg)](https://github.com/webmachinelearning/webnn-native/actions/workflows/memory_leak_check_dml.yml)
+[![Memory leak check - DirectMLX backend (Windows)](https://github.com/webmachinelearning/webnn-native/actions/workflows/memory_leak_check_dml.yml/badge.svg)](https://github.com/webmachinelearning/webnn-native/actions/workflows/memory_leak_check_dml.yml)
```

Comment:
@mingmingtasd Since you renamed `.github/workflows/build_test_dml.yml` file to `.github/workflows/build_test_dmlx.yml`, please also update URLs in README.md, thanks.

Reply:
Thanks, updated, PTAL again @BruceDai @fujunwei

Reply:
The CI failure is due to baseline. I have verified there is no regression. @BruceDai @fujunwei